(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[742],{1579:function(e,n,t){(window.__NEXT_P=window.__NEXT_P||[]).push(["/quickstart/deseq2",function(){return t(7489)}])},7489:function(e,n,t){"use strict";t.r(n),t.d(n,{__toc:function(){return f},default:function(){return _}});var a=t(5893),i=t(2673),s=t(8619),r=t(3257);t(7954);var o=t(2643),l=t(5675),d=t.n(l);function c(e){return(0,a.jsx)("div",{children:(0,a.jsx)("div",{style:{marginTop:"30px",display:"flex",justifyContent:"center"},children:(0,a.jsx)(d(),{src:e.path,width:"600",height:"600",alt:""})})})}t(7294);var p=t(948),h=t(1181),u=t(629),g=t(5861);let m=(0,p.ZP)(u.Z)(e=>{let{theme:n}=e;return{backgroundColor:"dark"===n.palette.mode?"#1A2027":"#fff",...n.typography.body2,padding:n.spacing(1),textAlign:"center",color:n.palette.text.secondary}});function x(){return(0,a.jsxs)(h.ZP,{container:!0,rowSpacing:1,columnSpacing:{xs:1,sm:2,md:3},children:[(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_vst.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"Variance Stabilizing Transformation (VST): After DESeq2 analysis, the VST is utilized for data quality evaluation through sample clustering and visualization."})]}),(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_dea2.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"DE results: MA plot illustrating differential gene expression analysis results obtained from DESeq2. The Y-axis represents the log fold change in gene expression between conditions, while the X-axis represents the average expression level across conditions."})]}),(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_GE_boxplot2.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"Gene BoxPlot: Gene expression box plot illustrates the distribution of expression levels if a gene across conditions post-DESeq2 analysis."})]}),(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_GE_heatmap2.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"Heatmap: Heatmap can be used to visualize the overall patterns of gene expression across samples or conditions following DESeq2 analysis."})]}),(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_volcano2.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"Volcano plot:A volcano plot is used to visualize the results of differential expression analysis by plotting the log fold change versus the statistical significance for each gene.."})]}),(0,a.jsxs)(h.ZP,{item:!0,xs:6,children:[(0,a.jsx)(m,{children:(0,a.jsx)(d(),{src:"/deseq2_venn1.png",width:"800",height:"800",alt:"hello"})}),(0,a.jsx)(g.Z,{variant:"caption",gutterBottom:!0,children:"Venn digaram: To visualize the overlap between different sets of differentially expressed genes (DEGs) across multiple conditions can provide insights into shared and condition-specific gene expression changes. ."})]})]})}let f=[{depth:2,value:"Deseq2 Shiny",id:"deseq2-shiny"},{depth:3,value:"Input Data Ttypes",id:"input-data-ttypes"},{depth:4,value:"Gene Counts Table",id:"gene-counts-table"},{depth:3,value:"Run deseq2",id:"run-deseq2"},{depth:4,value:"Init deseq2",id:"init-deseq2"},{depth:4,value:"Surrogate Variable analysis (SVA)",id:"surrogate-variable-analysis-sva"},{depth:3,value:"Deseq2 results",id:"deseq2-results"},{depth:4,value:"Visualization",id:"visualization"}];function q(e){let n=Object.assign({h2:"h2",p:"p",a:"a",strong:"strong",h3:"h3",h4:"h4",ul:"ul",li:"li"},(0,o.a)(),e.components);return(0,a.jsxs)(a.Fragment,{children:[(0,a.jsx)(n.h2,{id:"deseq2-shiny",children:"Deseq2 Shiny"}),"\n",(0,a.jsxs)(n.p,{children:["This Shiny app is a wrapper around ",(0,a.jsx)(n.a,{href:"https://bioconductor.org/packages/release/bioc/html/DESeq2.html",children:"Deseq2"}),", an R package for ",(0,a.jsx)(n.strong,{children:"differential gene expression analysis based on the negative binomial distribution"}),"."]}),"\n",(0,a.jsxs)(n.p,{children:["It is meant to provide an intuitive interface for researchers to easily upload, analyze, visualize, and explore ",(0,a.jsx)(n.strong,{children:"RNAseq count"})," data interactively with no prior programming knowledge in R."]}),"\n",(0,a.jsxs)(n.p,{children:["This tool supports ",(0,a.jsx)(n.strong,{children:"simple or multi-factorial"})," experimental design. It also allows for exploratory analysis when no replicates are available."]}),"\n",(0,a.jsxs)(n.p,{children:["The app also provides svaseq ",(0,a.jsx)(n.strong,{children:"Surrogate Variable Analysis"})," for hidden batch effect detection. The user can then include Surrogate Variables (SVs) as adjustment factors for downstream analysis (eg. differential expression)."]}),"\n",(0,a.jsxs)(n.p,{children:["For more information on svaseq, go to this ",(0,a.jsx)(n.a,{href:"https://www.bioconductor.org/packages/devel/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html#using-sva-with-deseq2",children:"link"})]}),"\n",(0,a.jsxs)(n.p,{children:["For details on how this package is used for RNASeq count data analysis and visualization, see ",(0,a.jsx)(n.a,{href:"https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html",children:"documentation"})]}),"\n",(0,a.jsx)(n.h3,{id:"input-data-ttypes",children:"Input Data Ttypes"}),"\n",(0,a.jsx)(n.p,{children:"The input file is a CSV file containing a raw gene count matrix generated by\nGeneCountMerger. You have the choice to either begin transcriptome analysis\ndirectly from this point or resume from the previous processing step with\nGeneCountMerger. If you've stored the count matrix with Ensemble IDs and gene names,\nyou can select the gene IDS and Gene Symbol option when uploading your data."}),"\n","\n",(0,a.jsx)(c,{path:"/deseq2_input.png"}),"\n",(0,a.jsx)(n.h4,{id:"gene-counts-table",children:"Gene Counts Table"}),"\n",(0,a.jsx)(n.p,{children:"After uploading the file, you will be able to view the matrix you uploaded.\nIf there are any duplicate names, they will be indicated with an ID appended to them,\nas highlighted in a red-colored message. For your convenience,\nthese duplicate names are displayed separately."}),"\n",(0,a.jsx)(c,{path:"/deseq2_table.png"}),"\n",(0,a.jsx)(n.h3,{id:"run-deseq2",children:"Run deseq2"}),"\n",(0,a.jsx)(n.h4,{id:"init-deseq2",children:"Init deseq2"}),"\n",(0,a.jsx)(n.p,{children:"To run deseq2 you must first specify the design formula, for more information about\nthe design formula click here."}),"\n",(0,a.jsx)(n.p,{children:"Click on ‘Initial DEseq2 to proceed’"}),"\n",(0,a.jsx)(c,{path:"/deseq2_init.png"}),"\n",(0,a.jsx)(n.h4,{id:"surrogate-variable-analysis-sva",children:"Surrogate Variable analysis (SVA)"}),"\n",(0,a.jsx)(n.p,{children:"After initialization parameters for deseq2 to run its optional to run Surrogate Variable\nanalysis (SVA)."}),"\n",(0,a.jsx)(c,{path:"/deseq2_run.png"}),"\n","\n",(0,a.jsx)(n.h3,{id:"deseq2-results",children:"Deseq2 results"}),"\n",(0,a.jsx)(n.p,{children:"Result of desq2 can be analysied by exploring newly from newly added tabs"}),"\n",(0,a.jsxs)(n.ul,{children:["\n",(0,a.jsx)(n.li,{children:"VST"}),"\n",(0,a.jsx)(n.li,{children:"DE result"}),"\n",(0,a.jsx)(n.li,{children:"Gene BoxPlot"}),"\n",(0,a.jsx)(n.li,{children:"And Heatmap"}),"\n",(0,a.jsx)(n.li,{children:"Volcano plot(Will be shown after the post anlysis of DE result)"}),"\n",(0,a.jsx)(n.li,{children:"Venn diagram(Will be shown after the post anlysis of DE result)"}),"\n"]}),"\n",(0,a.jsx)(n.h4,{id:"visualization",children:"Visualization"}),"\n",(0,a.jsx)(n.p,{children:"Various forms of visualizations are included"}),"\n",(0,a.jsx)(x,{})]})}let k={MDXContent:function(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},{wrapper:n}=Object.assign({},(0,o.a)(),e.components);return n?(0,a.jsx)(n,{...e,children:(0,a.jsx)(q,{...e})}):q(e)},pageOpts:{filePath:"pages/quickstart/deseq2.mdx",route:"/quickstart/deseq2",pageMap:[{kind:"Meta",data:{index:"Overview",quickstart:"Quickstart Guide",applications:"Applications",contact:{title:"Contact ↗",type:"page",href:"#",newWindow:!0},about:"About"}},{kind:"MdxPage",name:"about",route:"/about"},{kind:"Folder",name:"applications",route:"/applications",children:[{kind:"Meta",data:{data_preprocessing_and_qc:"Data preprocessing & QC",rna_seq:"RNA seq",meta_genomics:"Meta Genomics",epigenetics:"Epigenetics",varient_annotation:"Variant calling  and Annotation"}},{kind:"Folder",name:"data_preprocessing_and_qc",route:"/applications/data_preprocessing_and_qc",children:[{kind:"Meta",data:{gene_count_merger:"Gene Count Merger",merge_fpkms:"Merge FPKMS"}},{kind:"MdxPage",name:"gene_count_merger",route:"/applications/data_preprocessing_and_qc/gene_count_merger"},{kind:"MdxPage",name:"merge_fpkms",route:"/applications/data_preprocessing_and_qc/merge_fpkms"}]},{kind:"Folder",name:"epigenetics",route:"/applications/epigenetics",children:[{kind:"Meta",data:{bulk:"Bulk",single_cell:"Single Cell"}},{kind:"Folder",name:"bulk",route:"/applications/epigenetics/bulk",children:[{kind:"Meta",data:{atacseq_qc:"Atacseq QC"}},{kind:"MdxPage",name:"atacseq_qc",route:"/applications/epigenetics/bulk/atacseq_qc"}]}]},{kind:"Folder",name:"meta_genomics",route:"/applications/meta_genomics",children:[{kind:"Meta",data:{dada2:"DADA2",animalcules:"Animalcules"}},{kind:"MdxPage",name:"animalcules",route:"/applications/meta_genomics/animalcules"},{kind:"MdxPage",name:"dada2",route:"/applications/meta_genomics/dada2"}]},{kind:"Folder",name:"rna_seq",route:"/applications/rna_seq",children:[{kind:"Meta",data:{single_cell:"Single Cell",bulk_rna:"Bulk RNA"}},{kind:"Folder",name:"bulk_rna",route:"/applications/rna_seq/bulk_rna",children:[{kind:"Meta",data:{debrowser:"Debrowser",deseq2:"DEseq2",startapp:"STARTApp",enrichment:"Enrichment"}},{kind:"MdxPage",name:"debrowser",route:"/applications/rna_seq/bulk_rna/debrowser"},{kind:"MdxPage",name:"deseq2",route:"/applications/rna_seq/bulk_rna/deseq2"},{kind:"MdxPage",name:"enrichment",route:"/applications/rna_seq/bulk_rna/enrichment"},{kind:"MdxPage",name:"startapp",route:"/applications/rna_seq/bulk_rna/startapp"}]},{kind:"Folder",name:"single_cell",route:"/applications/rna_seq/single_cell",children:[{kind:"MdxPage",name:"monocle3",route:"/applications/rna_seq/single_cell/monocle3"},{kind:"MdxPage",name:"seuartV5",route:"/applications/rna_seq/single_cell/seuartV5"},{kind:"Meta",data:{monocle3:"Monocle3",seuartV5:"Seuartv5"}}]}]}]},{kind:"MdxPage",name:"index",route:"/"},{kind:"Folder",name:"quickstart",route:"/quickstart",children:[{kind:"MdxPage",name:"Installation",route:"/quickstart/Installation"},{kind:"Meta",data:{deseq2:"Example  RNAseq using DESeq2",Installation:"Installation"}},{kind:"MdxPage",name:"deseq2",route:"/quickstart/deseq2"}]},{kind:"MdxPage",name:"quickstart",route:"/quickstart"}],flexsearch:{codeblocks:!0},title:"Deseq2",headings:f},pageNextRoute:"/quickstart/deseq2",nextraLayout:s.ZP,themeConfig:r.Z};var _=(0,i.j)(k)},3257:function(e,n,t){"use strict";var a=t(5893);t(7294);let i={logo:(0,a.jsxs)(a.Fragment,{children:[(0,a.jsx)("img",{src:"/CGSB-Logo.png",width:"60"}),(0,a.jsx)("span",{style:{marginLeft:".4em",fontWeight:800},children:"NASQAR2 : Nucleic Acid SeQuence Analysis Resource"})]}),themeSwitch:{useOptions:()=>({light:"Light",dark:"Dark",system:"System"})},project:{link:"https://github.com/nabilr/nasqar2app"},chat:{link:"https://discord.com"},docsRepositoryBase:"https://github.com/nabilr/nasqar2app/tree/main",footer:{text:"Nextra Docs Template"},darkMode:!0,nextThemes:{defaultTheme:"light",forcedTheme:"light"}};n.Z=i},5789:function(){}},function(e){e.O(0,[692,481,143,888,774,179],function(){return e(e.s=1579)}),_N_E=e.O()}]);