## Deseq2 Shiny


This Shiny app is a wrapper around [Deseq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html), an R package for **differential gene expression analysis based on the negative binomial distribution**. 

It is meant to provide an intuitive interface for researchers to easily upload, analyze, visualize, and explore **RNAseq count** data interactively with no prior programming knowledge in R. 

This tool supports **simple or multi-factorial** experimental design. It also allows for exploratory analysis when no replicates are available.

The app also provides svaseq **Surrogate Variable Analysis** for hidden batch effect detection. The user can then include Surrogate Variables (SVs) as adjustment factors for downstream analysis (eg. differential expression). 

For more information on svaseq, go to this [link](https://www.bioconductor.org/packages/devel/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html#using-sva-with-deseq2)

For details on how this package is used for RNASeq count data analysis and visualization, see [documentation](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html)



### Input Data Ttypes

The input file is a CSV file containing a raw gene count matrix generated by
GeneCountMerger. You have the choice to either begin transcriptome analysis
directly from this point or resume from the previous processing step with
GeneCountMerger. If you've stored the count matrix with Ensemble IDs and gene names,
you can select the gene IDS and Gene Symbol option when uploading your data.

import {ImageSample} from '../../components/image_center'
import {ImageSample1} from '../../components/image_center'

<ImageSample1   path="/deseq2_input.png"/>



#### Gene Counts Table

After uploading the file, you will be able to view the matrix you uploaded. 
If there are any duplicate names, they will be indicated with an ID appended to them,
as highlighted in a red-colored message. For your convenience,
these duplicate names are displayed separately. 

<ImageSample1 path="/deseq2_table.png"/>

### Run deseq2

#### Init deseq2

To run deseq2 you must first specify the design formula, for more information about 
the design formula click here.

Click on ‘Initial DEseq2 to proceed’

<ImageSample1 path="/deseq2_init.png"/>

#### Surrogate Variable analysis (SVA)
After initialization parameters for deseq2 to run its optional to run Surrogate Variable
analysis (SVA).

<ImageSample1 path="/deseq2_run.png"/>



import MyGrid from '../../components/mygrid'

### Deseq2 results
Result of desq2 can be analysied by exploring newly from newly added tabs 
- VST
- DE result
- Gene BoxPlot
- And Heatmap
- Volcano plot(Will be shown after the post anlysis of DE result)
- Venn diagram(Will be shown after the post anlysis of DE result)

#### Visualization
Various forms of visualizations are included

<MyGrid/>