(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[603],{914:function(e,a,n){(window.__NEXT_P=window.__NEXT_P||[]).push(["/quickstart",function(){return n(9100)}])},9100:function(e,a,n){"use strict";n.r(a),n.d(a,{__toc:function(){return c}});var t=n(5893),i=n(2673),s=n(8619),r=n(3257);n(7954);var o=n(2643);let c=[];function l(e){let a=Object.assign({h1:"h1",p:"p"},(0,o.a)(),e.components);return(0,t.jsxs)(t.Fragment,{children:[(0,t.jsx)(a.h1,{children:"Nasqar Quick Start Tutorial"}),"\n",(0,t.jsx)(a.p,{children:"This tutorial offers insights into NASQAR, a cutting-edge web-based platform designed\nto enhance the analysis and visualization of high-throughput sequencing data, with a\nfocus on bioinformatics. NASQAR boasts a user-friendly interface conducive to interactive\ndata exploration, integrating a wide range of commonly used tools for tasks like\nTranscriptome Data Preprocessing, RNAseq Analysis (including Single-cell RNAseq),\nMetagenomics, and Gene Enrichment. By providing easy access to these powerful analysis tools,\nNASQAR aims to alleviate the computational challenges faced by researchers, enabling them to\neffectively analyze and interpret their sequencing data."})]})}let d={MDXContent:function(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},{wrapper:a}=Object.assign({},(0,o.a)(),e.components);return a?(0,t.jsx)(a,{...e,children:(0,t.jsx)(l,{...e})}):l(e)},pageOpts:{filePath:"pages/quickstart.mdx",route:"/quickstart",timestamp:1709040858e3,pageMap:[{kind:"Meta",data:{index:"Overview",quickstart:"Quickstart Guide",applications:"Applications",contact:{title:"Contact â†—",type:"page",href:"#",newWindow:!0},about:"About"}},{kind:"MdxPage",name:"about",route:"/about"},{kind:"Folder",name:"applications",route:"/applications",children:[{kind:"Meta",data:{data_preprocessing_and_qc:"Data preprocessing & QC",rna_seq:"RNA seq",meta_genomics:"Meta Genomics",epigenetics:"Epigenetics",varient_annotation:"Variant calling  and Annotation"}},{kind:"Folder",name:"data_preprocessing_and_qc",route:"/applications/data_preprocessing_and_qc",children:[{kind:"Meta",data:{gene_count_merger:"Gene Count Merger",merge_fpkms:"Merge FPKMS"}},{kind:"MdxPage",name:"gene_count_merger",route:"/applications/data_preprocessing_and_qc/gene_count_merger"},{kind:"MdxPage",name:"merge_fpkms",route:"/applications/data_preprocessing_and_qc/merge_fpkms"}]},{kind:"Folder",name:"epigenetics",route:"/applications/epigenetics",children:[{kind:"Meta",data:{bulk:"Bulk",single_cell:"Single Cell"}},{kind:"Folder",name:"bulk",route:"/applications/epigenetics/bulk",children:[{kind:"Meta",data:{atacseq_qc:"Atacseq QC"}},{kind:"MdxPage",name:"atacseq_qc",route:"/applications/epigenetics/bulk/atacseq_qc"}]}]},{kind:"Folder",name:"meta_genomics",route:"/applications/meta_genomics",children:[{kind:"Meta",data:{dada2:"DADA2",animalcules:"Animalcules"}},{kind:"MdxPage",name:"animalcules",route:"/applications/meta_genomics/animalcules"},{kind:"MdxPage",name:"dada2",route:"/applications/meta_genomics/dada2"}]},{kind:"Folder",name:"rna_seq",route:"/applications/rna_seq",children:[{kind:"Meta",data:{single_cell:"Single Cell",bulk_rna:"Bulk RNA"}},{kind:"Folder",name:"bulk_rna",route:"/applications/rna_seq/bulk_rna",children:[{kind:"Meta",data:{debrowser:"Debrowser",deseq2:"DEseq2",startapp:"STARTApp",enrichment:"Enrichment"}},{kind:"MdxPage",name:"debrowser",route:"/applications/rna_seq/bulk_rna/debrowser"},{kind:"MdxPage",name:"deseq2",route:"/applications/rna_seq/bulk_rna/deseq2"},{kind:"MdxPage",name:"enrichment",route:"/applications/rna_seq/bulk_rna/enrichment"},{kind:"MdxPage",name:"startapp",route:"/applications/rna_seq/bulk_rna/startapp"}]},{kind:"Folder",name:"single_cell",route:"/applications/rna_seq/single_cell",children:[{kind:"MdxPage",name:"monocle3",route:"/applications/rna_seq/single_cell/monocle3"},{kind:"MdxPage",name:"seuartV5",route:"/applications/rna_seq/single_cell/seuartV5"},{kind:"Meta",data:{monocle3:"Monocle3",seuartV5:"Seuartv5"}}]}]}]},{kind:"MdxPage",name:"index",route:"/"},{kind:"Folder",name:"quickstart",route:"/quickstart",children:[{kind:"MdxPage",name:"Installation",route:"/quickstart/Installation"},{kind:"Meta",data:{deseq2:"Example  RNAseq using DESeq2",Installation:"Installation"}},{kind:"MdxPage",name:"deseq2",route:"/quickstart/deseq2"}]},{kind:"MdxPage",name:"quickstart",route:"/quickstart"}],flexsearch:{codeblocks:!0},title:"Nasqar Quick Start Tutorial",headings:c},pageNextRoute:"/quickstart",nextraLayout:s.ZP,themeConfig:r.Z};a.default=(0,i.j)(d)},3257:function(e,a,n){"use strict";var t=n(5893);n(7294);let i={logo:(0,t.jsxs)(t.Fragment,{children:[(0,t.jsx)("img",{src:"/CGSB-Logo.png",width:"60"}),(0,t.jsx)("span",{style:{marginLeft:".4em",fontWeight:800},children:"NASQAR2 : Nucleic Acid SeQuence Analysis Resource"})]}),themeSwitch:{useOptions:()=>({light:"Light",dark:"Dark",system:"System"})},project:{link:"https://github.com/nabilr/nasqar2app"},chat:{link:"https://discord.com"},docsRepositoryBase:"https://github.com/nabilr/nasqar2app/tree/main",footer:{text:"Nextra Docs Template"},darkMode:!0,nextThemes:{defaultTheme:"light",forcedTheme:"light"}};a.Z=i},5789:function(){}},function(e){e.O(0,[692,888,774,179],function(){return e(e.s=914)}),_N_E=e.O()}]);